api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXTBQueries.java

```diff
@@ -0,0 +1,298 @@
+package org.openmrs.module.eptsreports.reporting.library.queries;
+
+import java.util.List;
+import org.apache.commons.lang.StringUtils;
+
+public class TXTBQueries {
+
+  /**
+   * Copied straight from INICIO DE TRATAMENTO ARV - NUM PERIODO: INCLUI TRANSFERIDOS DE COM DATA DE
+   * INICIO CONHECIDA (SQL) SqlCohortDefinition#91787a86-0362-4820-a4ee-025d5501198b in backup
+   *
+   * @return sql
+   */
+  public static String arvTreatmentIncludesTransfersFromWithKnownStartData(
+      Integer ARVPlanConceptId,
+      Integer startDrugsConceptId,
+      Integer historicalDrugsStartDateConceptId,
+      Integer ARTProgramId,
+      Integer pharmacyEncounterTypeId,
+      Integer artAdultFollowupEncounterTypeId,
+      Integer artPedFollowupEncounterTypeId) {
+    return "SELECT patient_id FROM (SELECT patient_id, Min(data_inicio) data_inicio "
+        + "FROM (SELECT p.patient_id, Min(e.encounter_datetime) data_inicio FROM patient p "
+        + "INNER JOIN encounter e ON p.patient_id = e.patient_id "
```

Comment:
@kaweesi we should be using `String.format()` on these queries to make them more readable and easier to copy to an SQL client when needed during investigations, development, etc. It's something that has been very useful in the past.

Reply:
thanks a lot @edrisse for your review, if i was making a choice on readability, then not using String.format() would be more readable to me. maybe easily copying the query but still u would deal with '+' ... etc

Reply:
@kaweesi I think you should change the code. We already have refactored the older reports to use `String.format()` and merging this code the way it is would be a step backwards.

Reply:
besides those 2 reasons, are there other reasons that were considered for the refactoring?

Reply:
Yes, we also started using the dimensions in a proper way (we were using composition queries) and moved the base cohort from the query definitions to the report definition. Also we now tend to use calculations when possible.